modules/cli/src/main/scala/scala/cli/commands/CompileOptions.scala

```diff
@@ -29,4 +37,16 @@ final case class CompileOptions(
 object CompileOptions {
   implicit lazy val parser: Parser[CompileOptions] = Parser.derive
   implicit lazy val help: Help[CompileOptions]     = Help.derive
+  implicit val customArgParser: ArgParser[Path] =
```

Comment:
We've not used a custom `ArgParser` for `os.Path` until now, we just accept an `Option[String]`, then convert its value to an `os.Path` with `os.Path(str, Os.pwd)`. That allows to filter out empty strings before the conversion in particular.

Maybe we could use an `ArgParser`, but it would need to be an `ArgParser[Option[os.Path]]`, and have empty strings (or empty after trimming) result in `None`. That can be done later.